requirements/all.txt

```diff
@@ -358,3 +217,208 @@ numpy==1.15.1 \
     --hash=sha256:df0b02c6705c5d1c25cc35c7b5d6b6f9b3b30833f9d178843397ae55ecc2eebb \
     --hash=sha256:e3660744cda0d94b90141cdd0db9308b958a372cfeee8d7188fdf5ad9108ea82 \
     --hash=sha256:f2362d0ca3e16c37782c1054d7972b8ad2729169567e3f0f4e5dd3cdf85f188e
+
+# Python Build Reasonableness for setuptools
+# TODO: Try removing
+# Code: https://github.com/openstack/pbr
+pbr==3.1.1 \
```

Comment:
``pbr`` doesn't appear to be used directly. It may patch setuptools in ways we need.

Reply:
IIRC this was a build dependency of something in the past, when you installed the library from a source dist. But I can’t remember anymore what that was.

Reply:
Thanks, that matches my vague memory of this module from past projects. I still see "Pabst Blue Ribbon", never "Python Build Reasonableness".